indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1636,7 +1639,8 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
             for item in self._display:
                 if item.input_type == 'accommodation':
                     if item.id in data:
-                        lp.append(Paragraph(data[item.id].friendly_data.get('choice').encode('utf-8'), text_format))
+                        choice = data[item.id].friendly_data.get('choice', '').encode('utf-8')
+                        lp.append(Paragraph(choice, text_format))
                         arrival_date = data[item.id].friendly_data.get('arrival_date')
                         arrival_date = format_date(arrival_date) if arrival_date else ''
```

Comment:
I think I got an error-report where there was such a keyerror.. anyway, probably after the PR with the modifications to the accommodation field, this is not needed.